### PR TITLE
WCAG replacing FAST

### PR DESCRIPTION
**Pull request by bbcjamesfletcher**
@anssiko this change is in response to https://github.com/webmachinelearning/ethical-webmachinelearning/issues/10

I'll outline the rationale in the issue thread, if every is happy can you review this?

***
<a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/ethical-webmachinelearning/pull/13.html" title="Last updated on May 3, 2022, 1:19 PM UTC (ffe92f1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/ethical-webmachinelearning/13/62a5765...ffe92f1.html" title="Last updated on May 3, 2022, 1:19 PM UTC (ffe92f1)">Diff</a>

### CODE REVIEW

**Comment by anssiko** [comment]
@michael-n-cooper does this PR look good to you? This is in response to the comments in issue #10. Thank you for your review and suggestions.

**Review by michael-n-cooper — CHANGES_REQUESTED** [review]
publisher for WCAG 2.1 should be "W3C Accessibility Guidelines Working Group", otherwise looks ok

File: index.bs [hunk 1]
```diff
@@ -25,6 +25,11 @@ Markup Shorthands: markdown yes
     "href": "https://w3c.github.io/apa/fast/checklist.html",
     "title": "Framework for Accessibility in the Specification of Technologies",
     "publisher": "W3C Accessible Platform Architectures Working Group"
+},
+  "WCAG": {
+    "href": "https://www.w3.org/TR/WCAG21",
+    "title": "Web Content Accessibility Guidelines (WCAG) 2.1",
+    "publisher": "W3C Accessible Platform Architectures Working Group"
```
Comment (anssiko) [inline index.bs]:
```suggestion
    "publisher": "W3C Accessibility Guidelines Working Group"
```

**Comment by anssiko** [comment]
Thanks @michael-n-cooper for your review. I've addressed your comment as proposed.

**Review by anssiko — APPROVED** [review]

**Pull request merged**